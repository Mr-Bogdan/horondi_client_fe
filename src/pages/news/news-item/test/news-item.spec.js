import React from 'react';
import NewsItem from '../news-item';

jest.mock('react-i18next', () => ({
  useTranslation: () => ({ t: () => null, i18n: { language: 'ua' } })
}));
jest.mock('../../news-item/news-item.style', () => ({ useStyles: () => ({}) }));
jest.mock('react-redux');

const props = {
  date: '',
  key: '',
  id: '',
  author: {},
  image: {},
  title: [],
  slug: '',
  text: []
};
describe('Test NewsItem', () => {
  it('should render component', () => {
    const component = shallow(<NewsItem {...props} />);
    expect(component).toBeDefined();
  });
});
