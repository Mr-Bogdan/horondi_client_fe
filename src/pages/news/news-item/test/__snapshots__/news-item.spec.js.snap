// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NewsItem should render component 1`] = `""`;
